packages/core/src/awsService/cloudWatchLogs/registry/liveTailSessionRegistry.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import { CLOUDWATCH_LOGS_LIVETAIL_SCHEME } from '../../../shared/constants'
+import { LiveTailSession, LiveTailSessionConfiguration } from './liveTailSession'
+import { ToolkitError } from '../../../shared'
+
+export class LiveTailSessionRegistry {
```

Comment:
You might be able to get away with extending NestedMap (from map.ts). It has the basic structure for a registry like this with get, set, delete already

Reply:
Seems like that works for us! Will change the implementation.